app/tests/integration/nimbus/utils/Dockerfile-rust-image

```diff
@@ -0,0 +1,45 @@
+FROM rust:buster AS builder
+
+ENV SQLCIPHER_LIB_DIR=/application-services/libs/desktop/linux-x86-64/sqlcipher/lib
+ENV SQLCIPHER_INCLUDE_DIR=/application-services/libs/desktop/linux-x86-64/sqlcipher/include
+ENV NSS_DIR=/application-services/libs/desktop/linux-x86-64/nss
+ENV NSS_STATIC=1
+
+RUN rustup target add x86_64-unknown-linux-musl
+RUN apt update && apt install -y musl-tools musl-dev
+RUN update-ca-certificates
+
+RUN apt-get update -qqy \
+  && apt-get -qqy install \
+    gyp \
+    ninja-build \
+    zlib1g-dev \
+    tclsh \
+    python3-venv
+
+RUN git clone https://github.com/mozilla/application-services.git
+
+RUN cd application-services \
+  && git submodule init \
+  && git submodule update --recursive \
+  && ./libs/verify-desktop-environment.sh
```

Comment:
So would it be better to just have it build on tags? I can cache the last version and build if there is a new one or something like that. Since this is on CI, I don't see an instance where we will specify a tag name, as that would require a whole commit, but locally, this makes sense.

Reply:
Idk if circleci has a comparable feature, but I know you can set variables in github actions, and change them in the settings without making a commit.

Reply:
circleci, its not as fancy as github actions 😅